package/Dockerfile.submariner-buildenv

```
@@ -0,0 +1,43 @@
+FROM fedora:33
```

Shouldn't "33" be an ARG by itself with a default of 33, this way we don't have to replace it in multiple places?
```suggestion
ARG FEDORA_VERSION
FROM fedora:${FEDORA_VERSION}
```

then in Makefile add `FEDORA_VERSION ?= 33`